spec/dry/logger/backends/proxy_spec.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+using RSpec::Support::HashString
```

Comment:
This might be the first refinement I've ever seen in the wild!

Reply:
I prefer it to global monkeypatches. Basically no downside if you restrict them to file or class scope.